DISCOVER/07_code_of_conduct.md

```diff
@@ -69,19 +69,18 @@ Make sure that **everyone involved in your conference/event** is aware that the
 **Why You Need a Code of Conduct**
 - [Why to have a CoC and how to make one](http://incisive.nu/2014/codes-of-conduct/)
 - [Why conferences need a code of conduct](https://jacobian.org/writing/codes-of-conduct/)
-- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)
+<!-- Citation: Rachel Nabors, "You Literally Cannot Pay Me to Speak Without a Code of Conduct," published September 1, 2015. Available at: http://rachelnabors.com/2015/09/01/code-of-conduct/ (Currently unavailable). -->
 - [Human Decency Is Not Enough: Why Cons Need Better Anti-Harassment Policies](https://www.wired.com/2013/07/convention-harassment-comic-con/)
 - [Should my tech conference community have a code of conduct and recommended resources](http://wunder.schoenaberselten.com/2016/02/17/should-my-tech-conference-community-have-a-code-of-conduct-recommended-resources/)
 
 **Crafting a Code of Conduct**
 - [A CoC builder](http://codeofconduct.io)
 - [Conference Code of Conduct](http://confcodeofconduct.com/)
 - [Codes of Conduct 101 + FAQ](https://www.ashedryden.com/blog/codes-of-conduct-101-faq)
-- [How to design a code of conduct for your community](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/)
+- [How to design a code of conduct for your community](https://web.archive.org/web/20201026215352/https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/)
 - [How Tech Codes of Conduct Fail](https://medium.com/@gusseting/tech-codes-of-conduct-e4e05c6f539f)
 
 **Managing a Code of Conduct**
 - [NYC PyLadies Meetup: Managing our Code of Conduct](https://reshamas.github.io/managing-our-code-of-conduct/)
-- [Beyond Guidelines: Setting the stage for good behavior in your community](http://cmxhub.com/article/beyond-guidelines-setting-the-stage-for-good-behavior-in-your-community/)
+<!-- Citation: "Beyond Guidelines: Setting the Stage for Good Behavior in Your Community," CMX Hub. Available at: http://cmxhub.com/article/beyond-guidelines-setting-the-stage-for-good-behavior-in-your-community/ (Currently unavailable). -->
```

Comment:
Same comment as above, would recommend we use[ markdown footnotes](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/) rather than HTML comments.